portfolio/src/main/webapp/index.html

```diff
@@ -98,5 +144,63 @@ <h1>Work Experience</h1>
           <h1>Contact Me</h1>
       </section>
     </div>
+
+    <!-- Lightbox Gallery -->
+          <div id="lightbox">
+              <a class="interface-buttons" id="close" onclick="closeLightbox();">Close X</a>
+              <div class="lightbox-content">
+                  <a class="interface-buttons" id="prev" onclick="changePic(-1)">&#10094;</a>
+                  <a class="interface-buttons" id="next" onclick="changePic(1)">&#10095;</a>
+                  <div class="lightbox-pic">
+                      <img src="/images/arlo1.jpg" alt="Arlo chilling in driveway">
```

Comment:
You could also consider using `<figure>` and `<figcaption>` for these images: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption

Reply:
Thanks!!